code_folder/C0_data_processing.py

```diff
@@ -57,27 +57,79 @@ def simulation_settings(simulation_settings):
 
     def process_all_assets(dict_values):
         # read timeseries with filename provided for technical parameters (efficiency and minimum and maximum storage level)
+
         for asset in dict_values["energyConversion"]:
             dict_asset = dict_values["energyConversion"][asset]
-            if "file_name" in dict_asset["efficiency"]:
+            # in case there is one value provided (case one input bus and one output bus)
+            if isinstance(dict_asset["efficiency"]["value"], dict):
                 helpers.receive_timeseries_from_csv(
                     dict_values["simulation_settings"], dict_asset, "efficiency"
                 )
-
+            # in case there are more than one values provided (case n input busses and 1 output bus or 1 input bus and n output busses)
+            # dictionaries with filenames and headers will be replaced by timeseries, scalars will be mantained
+            elif isinstance(dict_asset["efficiency"]["value"], list):
+                updated_values = []
+                values_info = (
+                    []
+                )  # filenames and headers will be stored to allow keeping track of the timeseries generation
+                for element in dict_asset["efficiency"]["value"]:
+                    if isinstance(element, dict):
+                        updated_values.append(
+                            helpers.get_timeseries_multiple_flows(
+                                dict_values["simulation_settings"],
+                                dict_asset,
+                                element["file_name"],
+                                element["header"],
+                            )
+                        )
+                        values_info.append(element)
+                    else:
+                        updated_values.append(element)
+                dict_asset["efficiency"]["value"] = updated_values
+                if len(values_info) > 0:
+                    dict_asset["efficiency"].update({"values_info": values_info})
+
+        # same distinction of values provided with dictionaries (one input and one output) or list (multiple).
+        # The can at turn be scalars, mantained, or timeseries
```

Comment:
```suggestion
        # The can in turn be scalars, mantained, or timeseries
```
What does "mantained" mean here?